assets/mails/dining/dining_list_deleted.html

```diff
@@ -6,19 +6,51 @@
         Hey {{ user.first_name }}
     </p>
     <p>
-        Sorry to be the bringer of bad news, but...<br>
-        {{cancelled_by}} has <b>cancelled</b> the {{ dining_list.association.shorthand }}dining list on {{dining_list.date}}. <br>
-        As a result you may need to find
-        a meal elsewhere. Maybe you can join another dining list or start your own. You can check it here:
+        We regret to inform you that the following dining list has been <b>cancelled</b>.
+    </p>
+    <table>
+        <tr><td>
+            Date
+        </td><td>
+            {{dining_list.date}}
+        </td></tr>
+        <tr><td>
+            Dish
+        </td><td>
+            {{dining_list.dish}}
+        </td></tr>
+        <tr><td>
+            By
+        </td><td>
+            {{dining_list.claimed_by}}
+        </td></tr>
+        <tr><td>
+            Association
+        </td><td>
+            {{dining_list.association}}
+        </td></tr>
+
+    </table>
+    <p>
+        As you were subscribed to this dining list, you have been removed from the dining list and your money has been refunded
+        Maybe you can join or start another dining list. You can do so here:
     </p>
     <div>
         <a href="{{ day_view_url }}"
            style="background-color: #375a7f;padding: 0.75em; border-radius: 0.25rem; color: white; text-decoration: none;">
             Go to day view</a>
     </div>
 
-    <p style="padding-top: 1em">
-        Enjoy your day
+    <p>
+        We are sorry for the inconvenience,
+    </p>
+    <p>
+        With kind regards,
+    </p>
+    <p style="padding-top: 1em;">
+
+
+        The Scala Dining App
```

Comment:
I actually really like the tone of this message :smile:, only the 'Hey Maarten' is kinda out of tone, probably 'Hi Maarten' is better.